module/VuFind/src/VuFind/Search/Base/Options.php

```diff
@@ -320,6 +320,28 @@ abstract class Options implements TranslatorAwareInterface
      */
     protected $firstlastNavigation = false;
 
+    /**
+     * Should HierarchicalFacetFilters and HierarchicalExcludeFilters
+     * apply in advanced search
+     *
+     * @var bool
+     */
+    protected $filterFacetsInAdvanced = false;
+
+    /**
+     * Excluded filters
+     *
+     * @var array
+     */
+    protected $excludeFilters = [];
```

Comment:
Again, probably would benefit from more specific naming if we're not going to use this for non-hierarchical filters too.